packages/repository-json-schema/test/unit/build-schema.unit.ts

```diff
@@ -8,10 +8,14 @@ import {isComplexType, stringTypeToWrapper, metaToJsonProperty} from '../..';
 
 describe('build-schema', () => {
   describe('stringTypeToWrapper', () => {
-    it('returns respective wrapper of number, string and boolean', () => {
+    it('returns respective wrapper of number, string, boolean, array, buffer, date and object', () => {
```

Comment:
One (logical) assert per test - please refactor this test into a series of small test cases.

```ts
it('returns String for "string"', () => {
  expect(stringTypeToWrapper('string')).to.eql(String);
});

// ...
```